bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -707,4 +742,15 @@ public synchronized void close() throws IOException {
   public BigtableOptions getOptions() {
     return this.options;
   }
+
+  /**
+   * Getter for the field <code>gcjClientContext</code>.
+   *
+   * @return a HashMap of Connection Name (String) to {@link ClientContext} object.
+   */
+  public HashMap<String, ClientContext> getGCJClientContext() {
```

Comment:
Does this need to be public? If so, can you mark this as `@InternalApi`. See this PR for an example:#2245

Reply:
I think you should just inline the getter. I don't think it brings much value here